README.md

```diff
@@ -1,6 +1,15 @@
 ![VegBank](http://vegbank.org/vegbank/images/vegbank_logo69x100trans.gif)
 ![VegBank](http://vegbank.org/vegbank/images/vegbank_caps170x40outline.jpg)
 
+
+## VegBank
+
+- **Authors**: Last, First (ORCID); ...
+- **License**: [Apache 2](http://opensource.org/licenses/Apache-2.0)
+- [Package source code on GitHub](https://github.com/NCEAS/vegbank2)
+- [**Submit Bugs and feature requests**](https://github.com/NCEAS/vegbank2/issues)
+- Contact us: support@dataone.org
```

Comment:
Can/should we use an apporpriate VegBank email list?

Reply:
Changed to help@vegbank.org for now.